internal/handlers/common/params.go

```diff
@@ -186,7 +204,14 @@ func GetSkipStageParam(value any) (int64, error) {
 
 // getBinaryMaskParam matches value type, returning bit mask and error if match failed.
 // Possible values are: position array ([1,3,5] == 010101), whole number value and types.Binary value.
-func getBinaryMaskParam(mask any) (uint64, error) {
+//
+// Returned errors:
+//   - ErrBadValue if positional argument is not a number;
+//   - ErrBadValue if positional argument is negative;
+//   - ErrBadValue if bitmask argument is not a whole number;
```

Comment:
```suggestion
```
I think we are not returning `ErrBadValue` for non-whole number. We return `ErrFailedToParse`

Reply:
Removed that line.